containers/docker/README.md

```diff
@@ -0,0 +1,53 @@
+# HPCC Platform Community Version
+  ubuntu (bionic)/
+  -  base/<version>/Dockerfile: HPCC Platform Community version prequisites
+     Docker Hub:  hpccsystems/hpcc-base
+  -  platform/<version>/Dockerfile  HPCC Platform Community Docker build file
+     Docker Hub:  hpccsystems/platform
+  -  clienttools/<version>Dockerfile: HPCC Clienttools Community Docker build file
+     Docker Hub: hpccsystems/clienttools
+
+## How to buid
+Create HPCC Platform Docker image :
+```console
+sudo docker build -t hpccsystems/platform:7.4.8-1 --build-arg version=7.4.8-1 .
+```
+
+
+# HPCC Platform Internal Version
+  Reference LN container/docker/
+
+# Development and custom Build
+  <bionic|disco|eoan>/
+  -   base/Dockerfile: HPCC Platform community version prequisites
+  -   dev/<version>/Dockerfile: Build Server Docker build file
+  -   platform/<version>/Dockerfile: Docker build file for compiling and building HPCC Platform image
+  -   clienttools/<version>/Dockerfile: Docker build file for compiling and building HPCC Clienttools image
+
+  gcc/<7|8|9>/
+  -   base/Dockerfile: HPCC Platform community version prequisites
+  -   dev/<version>/Dockerfile: Build Server Docker build file
+  -   platform/<version>/Dockerfile: Docker build file for compiling and building HPCC Platform image
+
+## How to build
+  The compiling and building Docker build file has "ARG" for various input parameter.
```

Comment:
Compiling and building the Docker build file has an "ARG" for various input parameters.